db/init.sql

```diff
@@ -0,0 +1,41 @@
+ CREATE TABLE IF NOT EXISTS subscription(
+    subscription_id SERIAL PRIMARY KEY,
+    name  VARCHAR(100) NOT NULL,
+    duration INTERVAL NOT NULL,
+    cost INTEGER NOT NULL
+    ); 
+
+
+CREATE TABLE IF NOT EXISTS users(
+    user_id SERIAL PRIMARY KEY,
+    username VARCHAR(50)  NOT NULL,
+    age INTEGER NOT NULL,
+    email_address VARCHAR(100) NOT NULL,
+    password VARCHAR(100) NOT NULL,
+    is_admin BOOLEAN DEFAULT FALSE ,
+    subscription_id INTEGER  REFERENCES subscription(subscription_id),
+    subscription_end_date DATE 
+);
+CREATE TABLE IF NOT EXISTS book(
+    book_id SERIAL PRIMARY KEY,
```

Comment:
keep it as just id, if the id is of the same table. Only when referring as foreign key, we need to name like this.